src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -592,6 +592,14 @@ private function addService($id, $definition)
             $return[] = sprintf("@throws InactiveScopeException when the '%s' service is requested while the '%s' scope is not active", $id, $scope);
         }
 
+        if ($definition->isDeprecated()) {
+            if ($return && 0 === strpos($return[count($return) - 1], '@return')) {
+                $return[] = '';
+            }
+
+            $return[] = '@deprecated';
```

Comment:
I'm sorry but I don't see the blocker for putting the default message here. It wouldn't be a mistake, and we provide a way to have a better one.
If I go up to your arguments, then I would recommend to remove the default message completely and force deprecation authors to always provide one. That being for this use case or for the trigger_error below, or anywhere else we use the message.